Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -4273,3 +4275,30 @@ std::variant<bool, std::array<CVector, 4>> CLuaVehicleDefs::OOP_GetVehicleEntryP
 
     return entryPoints;
 }
+
+bool CLuaVehicleDefs::AddVehicleSirens(CClientVehicle* vehicle, unsigned char ucSirenType, unsigned char ucSirenCount,  std::optional<bool> bEnable360,  std::optional<bool> bEnableLOSCheck ,  std::optional<bool> bEnableRandomiser,  std::optional<bool> bEnableSilent)
+{
+    eClientVehicleType vehicleType = vehicle->GetVehicleType();
+
+    if (vehicleType != CLIENTVEHICLE_PLANE && vehicleType != CLIENTVEHICLE_BOAT && vehicleType != CLIENTVEHICLE_TRAILER && vehicleType != CLIENTVEHICLE_HELI &&
+          vehicleType != CLIENTVEHICLE_BIKE && vehicleType != CLIENTVEHICLE_BMX)
```

Comment:
It will be more readable to simply
```cpp
if (vehicleType != CCLIENTVEHICLE_AUTOMOBILE && vehicleType != CCLIENTVEHICLE_BIKE)
    return false;
```

Reply:
do you mean CLIENTVEHICLE_CAR 
there is no such thing as CCLIENTVEHICLE_AUTOMOBILE

```c++
enum eClientVehicleType
{
    CLIENTVEHICLE_NONE,
    CLIENTVEHICLE_CAR,
    CLIENTVEHICLE_BOAT,
    CLIENTVEHICLE_TRAIN,
    CLIENTVEHICLE_HELI,
    CLIENTVEHICLE_PLANE,
    CLIENTVEHICLE_BIKE,
    CLIENTVEHICLE_MONSTERTRUCK,
    CLIENTVEHICLE_QUADBIKE,
    CLIENTVEHICLE_BMX,
    CLIENTVEHICLE_TRAILER
};
```

Reply:
Yes, my mistake. Maybe it's also worth adding a bike, quad and monster_truck? Does the server side support this?

Reply:
it only support cars

Reply:
```
noSirenLights = {
	400, 401, 402, 403, 404, 405, 406, 417, 420, 423, 425, 430, 432, 433, 435, 438, 441, 446, 447, 448, 452, 453, 454, 460, 461, 462, 463, 464,
	465, 468, 469, 472, 473, 476, 481, 484, 487, 488, 493, 497, 501, 509, 510, 511, 512, 513, 519, 520, 521, 522, 523, 528, 537, 538, 539, 544,
	548, 553, 563, 564, 569, 570, 577, 581, 586, 590, 592, 593, 595, 600, 601, 602, 603, 604, 605, 606, 607, 608, 610, 611
} 
``` those are not supported  https://wiki.multitheftauto.com/wiki/Vehicle_IDs#Lua_table_of_vehicles_that_doesn.27t_support_siren_lights

Reply:
So quad and monster trucks supports sirens